README.md

```diff
@@ -1,29 +1,26 @@
 # Airship Plugin
 
-Preview of the Airship Flutter plugin.
+Airship Flutter plugin allows using Airship's native iOS and Android APIs with Flutter apps written in Dart.
 
-This plugin is currently in development or published. APIs will most likely change before the first
-major release.
+## Usage
 
-## Status
+1. Add the airship dependency to your package's pubspec.yaml file:
 
-Feature Status:
-- [x] Tags
-- [x] Notifications
-- [x] Deep links
-- [x] Inbox
-- [x] Named User
-- [x] Push Events
-- [ ] Custom events
-- [ ] Notification management
-- [ ] Tag Groups
+```
+dependencies:
+  airship: ^1.0.0
+```
 
-## Usage
+2. Install your flutter package dependencies by running the following in the command line at your project's root directory:
 
-This plugin is not yet published, so you will have to use the current master repo as a [dependency in your pubspec.yaml file](https://flutter.io/platform-plugins/).
 ```
-  airship:
-    git: git://github.com/urbanairship/airship-flutter.git
+$ flutter pub get
+```
+
+3. Import airship into your project:
```

Comment:
Is there a particular source file this needs to go in? Maybe it's obvious to Flutter developers, or maybe more detail would be helpful.

Reply:
no, this is the only one, there is also an example below with this re-iterated